gamemode/modules/base/sv_gamemode_functions.lua

```diff
@@ -94,6 +94,18 @@ function GM:canSeeLogMessage(ply, message, colour)
     return true
 end
 
+function GM:canEarnNPCMoney(ply, npc)
+    return GAMEMODE.Config.npckillpay > 0
+end
+
+function GM:calculateNPCMoney(ply, npc)
+    -- A NPC spawned by an addon might be worth more money than the default
+    if npc.KillValue then
+        return npc.KillValue
```

Comment:
I'm not too sure about this var being in DarkRP by default. It sounds generic enough to mean something else entirely in another addon. 
It's also obscure enough for people not to know it exists until they read the code. With the hook support you've added this functionality can go into your addon. 

What do you think?

Reply:
(I'm happy to rename the variable or drop the commit if you prefer though)